sites/hn.svelte.dev/src/routes/user/[name]/+page.js

```diff
@@ -1,7 +1,5 @@
 export const csr = false;
 
-/** @type {import('./$types').PageLoad} */
 export async function load({ params, fetch }) {
-	const res = await fetch(`https://api.hnpwa.com/v0/user/${params.name}.json`);
-	return res.json();
+	return fetch(`https://api.hnpwa.com/v0/user/${params.name}.json`).then((r) => r.json());
```

Comment:
making it one line makes sense, but why not use `await` still?
```suggestion
	return (await fetch(`https://api.hnpwa.com/v0/user/${params.name}.json`)).json();
```

Reply:
I dunno, await has felt weird when surrounded by brackets. .then feels cleaner for this case somehow

Reply:
What happens if invalid JSON is returned from the API? I think the two ways of writing it would behave differently, but I'm not exactly sure what the behavior of the `.then()` version is

Reply:
It's the same. If it errors, then .json() errors out, giving the same error as await version